### DISCUSSION TITLE
Github registration throws errors

### DISCUSSION THREAD

**Discussion by dkiia** [lsc-hubs/kenya-hub#17] [General]
The process of giving feedback via GIThub is not easy. cos it gives '' error'' state has expired.

**Comment by pvgenuchten** [comment]
Hi @dkiia, sorry to hear you had challenges with github registration. Good to see you have succeeded anyway!

In my opinion we need to apply authentication before contributing to the hub. Github is one of the options, google/microsoft/meta or a Kenian identity provider could be an alternative. You have a favourite one?
